.github/workflows/test.yml

```
@@ -0,0 +1,51 @@
+name: Run unit and style tests
+
+on:
+  pull_request:
+  push:
+    branches:
+      - develop
+      - master
+  # Allow manually triggering the workflow.
+  workflow_dispatch:
+
+jobs:
+  test:
+    name: Run tests on PHP ${{ matrix.php }}
+    runs-on: ubuntu-latest
+    continue-on-error: ${{ matrix.experimental == true }}
+
+    strategy:
+      matrix:
+        php:
+          - '5.4'
+          - '5.5'
+          - '5.6'
+          - '7.0'
+          - '7.1'
+          - '7.2'
+          - '7.3'
+          - '7.4'
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php }}
+          ini-values: error_reporting=E_ALL, display_errors=On
```

```suggestion
          ini-values: zend.assertions=1, error_reporting=-1, display_errors=On
```

* `E_ALL` does not contain **_all_** errors on all PHP versions, `-1` does.
* Assertions _may_ be in use in the tests, so never a bad thing to enable them when the CI includes a test run.